### PR TITLE
Add guidelines for Less/Sass operators

### PR DESCRIPTION
**Pull request by sgarrity**
As discussed in issue #129

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -330,6 +331,16 @@
   </div>
 </div>
 
+<div class="section" id="css-operators">
+  <div class="col">
+    <h3>Operators Less and Sass</h3>
+    <p>When using math operators in Less and Sass in properties with multiple values (margin, padding, etc.), wrap math operations in brackets and separate values, variables, and operators with a single space.</p>
```
Comment (mdo) [inline index.html]: Maybe something like:

> For improved readability, wrap all math operations in parentheses with a single space between values, variables, and operators.

Short and sweet :).

Comment (sgarrity) [inline index.html]: > Short and sweet :).

Update! Should I squash the commits?

Comment (mdo) [inline index.html]: Nah, all good :).

**Comment by mdo** [comment]
Thanks btw!

**Pull request merged**